r-package/survlearners/R/slasso_surv.R

```diff
@@ -87,7 +87,7 @@ slasso_surv = function(x, w, y, D, times,
                              standardize = FALSE,
                              alpha = alpha)
 
-  s_beta <- as.vector(t(coef(s_fit, s = lambda_choice)))
+  s_beta <- as.vector(t(as.matrix(coef(s_fit, s = lambda_choice))))
```

Comment:
Why can't this just be `s_beta <- as.vector(coef(s_fit, s = lambda_choice))`?

Reply:
because I want a row vector instead of a column vector for later use. Also trying to change follow my original code that worked for my simulations